### PR TITLE
chore: Updated all references to internal api docs and point to the g…

### PR DESCRIPTION
**Pull request by bizob2828**
…ithub hosted links. Also removed redundant snippets with links to robust example applications

Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

* What problems does this PR solve?

We had fragmented documentation around using the Node.js agent.  The Node.js agent team has spent a lot of time building out examples.  This PR updates most code snippets with links to functioning example applications.

Also, we had redundant information around the Node.js agent API. We host an [API docs](https://newrelic.github.io/node-newrelic/) site and it gets updated on every release.  This is the most recent information and any information documenting statically in docs-website could get out of date. This PR also updates any links to API methods to use the `newrelic.github.io` links.

## Links
Closes https://github.com/newrelic/node-newrelic/issues/2468

### CODE REVIEW

File: src/content/docs/apm/agents/nodejs-agent/extend-your-instrumentation/nodejs-v1-custom-instrumentation-legacy.mdx [hunk 1]
```diff
@@ -1,145 +0,0 @@
----
```
Comment (rhetoric101) [inline src/content/docs/apm/agents/nodejs-agent/extend-your-instrumentation/nodejs-v1-custom-instrumentation-legacy.mdx:1]: @bizob2828 it looks like there's still a reference to this file in the left navigation.

image

Lines 509 and 510 should be removed from `src/nav/apm.yml`

```
              - title: Node.js v1 custom instrumentation
                path: /docs/apm/agents/nodejs-agent/extend-your-instrumentation/nodejs-v1-custom-instrumentation-legacy
```

File: src/content/docs/apm/agents/nodejs-agent/installation-configuration/install-nodejs-agent.mdx [hunk 2]
```diff
@@ -98,7 +98,7 @@ To install the Node.js agent:
...
 
-     If neither of these options work for you, (e.g. asynchronously loading api keys from a remote location during application bootstrapping), you may also add stock instrumentation to an already loaded [supported module](https://github.com/newrelic/node-newrelic/blob/0113eb5f0e707dc662a17d262a841503bab88841/lib/instrumentations.js#L6#L6) by using [`newrelic.instrumentLoadedModule`](/docs/apm/agents/nodejs-agent/api-guides/nodejs-agent-api/#instrumentLoadedModule):
+     If neither of these options work for you, (e.g. asynchronously loading api keys from a remote location during application bootstrapping), you may also add stock instrumentation to an already loaded [supported module](https://github.com/newrelic/node-newrelic/blob/0113eb5f0e707dc662a17d262a841503bab88841/lib/instrumentations.js#L6#L6) by using [`newrelic.instrumentLoadedModule`](https://newrelic.github.io/node-newrelic/API.html#instrumentLoadedModule):
```
Comment (rhetoric101) [inline src/content/docs/apm/agents/nodejs-agent/installation-configuration/install-nodejs-agent.mdx:101]:
```suggestion
     If neither of these options work for you, (for example, asynchronously loading API keys from a remote location during application bootstrapping), you may also add stock instrumentation to an already loaded [supported module](https://github.com/newrelic/node-newrelic/blob/0113eb5f0e707dc662a17d262a841503bab88841/lib/instrumentations.js#L6#L6) by using [`newrelic.instrumentLoadedModule`](https://newrelic.github.io/node-newrelic/API.html#instrumentLoadedModule):
```

File: src/content/docs/apm/agents/nodejs-agent/installation-configuration/update-nodejs-agent.mdx [hunk 3]
```diff
@@ -475,7 +475,7 @@ Before upgrading to Node.js version 7, review this information for major changes
       </td>
 
       <td>
-        * Applications must use the configuration [`api.custom_attributes_enabled` or the associated environment variable `NEW_RELIC_API_CUSTOM_ATTRIBUTES`](/docs/agents/nodejs-agent/installation-configuration/nodejs-agent-configuration/#custom-attributes) to enable [`newrelic.addCustomAttribute`](/docs/agents/nodejs-agent/api-guides/nodejs-agent-api/#add-custom-attribute) and [`newrelic.addCustomAttributes`](/docs/agents/nodejs-agent/api-guides/nodejs-agent-api/#add-custom-attributes).
+        * Applications must use the configuration [`api.custom_attributes_enabled` or the associated environment variable `NEW_RELIC_API_CUSTOM_ATTRIBUTES`](/docs/agents/nodejs-agent/installation-configuration/nodejs-agent-configuration/#custom-attributes) to enable [`newrelic.addCustomAttribute`](https://newrelic.github.io/node-newrelic/API.html#addCustomAttribute) and [`newrelic.addCustomAttributes`](https://newrelic.github.io/node-newrelic/API.html#addCustomAttributes).
```
Comment (rhetoric101) [inline src/content/docs/apm/agents/nodejs-agent/installation-configuration/update-nodejs-agent.mdx:478]:
```suggestion
        * To enable [`newrelic.addCustomAttribute`](https://newrelic.github.io/node-newrelic/API.html#addCustomAttribute) and [`newrelic.addCustomAttributes`](https://newrelic.github.io/node-newrelic/API.html#addCustomAttributes), applications must use the configuration [`api.custom_attributes_enabled` or the associated environment variable `NEW_RELIC_API_CUSTOM_ATTRIBUTES`](/docs/agents/nodejs-agent/installation-configuration/nodejs-agent-configuration/#custom-attributes).
```

File: src/content/docs/apm/agents/nodejs-agent/extend-your-instrumentation/nodejs-v1-custom-instrumentation-legacy.mdx (hunk 1 above)
Comment (bizob2828) [reply src/content/docs/apm/agents/nodejs-agent/extend-your-instrumentation/nodejs-v1-custom-instrumentation-legacy.mdx:1]: ah ok. ill do that

**Review by rhetoric101 — APPROVED** [review]
@bizob2828 thanks for making these API updates!

**Comment by bizob2828** [comment]
> @bizob2828 thanks for making these API updates!

Thanks for reviewing this @rhetoric101!

**Pull request merged**